### ISSUE TITLE
Installing OSS 7.5.0-beta2 on windows succeeds but logs out lock error

### ISSUE THREAD

**Issue by hugohaggmark** [grafana/grafana#32166]
**What happened**:
![image](https://user-images.githubusercontent.com/562238/111797987-ec592000-88c9-11eb-8d9b-c0c4e7040096.png)
![image (1)](https://user-images.githubusercontent.com/562238/111797989-ee22e380-88c9-11eb-9e04-4199cd378be5.png)

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana Image Renderer version: 2.0.1
- Grafana version: 7.5.0-beta2
- Installed plugin or remote renderer service:
- OS Grafana Image Renderer is installed on:
- User OS & Browser: Chrome, Window 10
- Others:

**Comment by dprokop** [comment]
Same thingy with Enterprise. I assume it's not OSS/Enterprise related

**Comment by marefr** [comment]
Tested on 7.5.x branch using linux and successfully can render images given that I've properly installed the plugin by first stopping Grafana, installing it and starting Grafana:
```
~/go/src/github.com/grafana/grafana on v7.5.x *3 ?2                                           
❯ ./bin/grafana-cli plugins install grafana-image-renderer 
installing grafana-image-renderer @ 2.0.1
from: https://grafana.com/api/plugins/grafana-image-renderer/versions/2.0.1/download
into: /home/marcus/go/src/github.com/grafana/grafana/data/plugins

Error: ✗ failed to extract plugin archive: failed to extract file: file "/home/marcus/go/src/github.com/grafana/grafana/data/plugins/grafana-image-renderer/plugin_start_linux_amd64" is in use - please stop Grafana, install the plugin and restart Grafana

~/go/src/github.com/grafana/grafana on v7.5.x *3 ?2                                 took 2m 6s
❯ ./bin/grafana-cli plugins install grafana-image-renderer
installing grafana-image-renderer @ 2.0.1
from: https://grafana.com/api/plugins/grafana-image-renderer/versions/2.0.1/download
into: /home/marcus/go/src/github.com/grafana/grafana/data/plugins

✔ Installed grafana-image-renderer successfully 

Restart grafana after installing plugins . <service grafana-server restart>
```

**Comment by marefr** [comment]
This shouldn't be a blocker for 7.5 stable so will remove this from 7.5.0 milestone for now. But lets verify when 7.5 has been released

**Comment by marefr** [comment]
@ying-jeanne think you can investigate this when times allow? Seems to be a windows issue

**Comment by jessabe** [comment]
We believe that this issue might be fixed as of 8.1 -- as of #37367. Please re-open if this is not the case. Closing for now. :)
